applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -342,22 +370,44 @@ static void syncable_cb(struct bt_bap_broadcast_sink *sink, bool encrypted)
 	strncpy(bis_encryption_key, CONFIG_BT_AUDIO_BROADCAST_ENCRYPTION_KEY, 16);
 #endif /* (CONFIG_BT_AUDIO_BROADCAST_ENCRYPTED) */
 
-	if (!playing_state) {
-		LOG_DBG("Syncable received, but either in paused_state or already in a stream");
+	if (active_stream.stream != NULL && active_stream.stream->ep != NULL) {
+		if (active_stream.stream->ep->status.state == BT_BAP_EP_STATE_STREAMING) {
+			LOG_WRN("Syncable received, but already in a stream");
```

Comment:
So syncable_cb will not be called if we have an endpoind that is in BT_BAP_EP_STATE_STREAMING?
If syncable_cb is still called, this should probably not be a WRN

Reply:
Discuss

Reply:
Discussed-OK as it is.